bdd/steps/discount_transaction_steps.py

```diff
@@ -248,6 +248,11 @@ def step_check_latest_cancellation_failed(context):
     assert context.latest_cancellation_response.status_code >= 400
 
 
+@then('the latest cancellation by citizen fails')
+def step_check_latest_cancellation_by_citizen_failed(context):
+    assert context.latest_citizen_cancellation_response.status_code >= 400
```

Comment:
@Benedetta-fabbri can we assert the exact expected status code returned?
Probably this would result in more than one method for cancellation failures but we have more control over API interface changes.